SCClassLibrary/Common/Control/Server.sc

```diff
@@ -230,6 +233,9 @@ ServerOptions {
 		if (thisProcess.platform.name === \osx && Server.program.asString.endsWith("supernova").not && safetyClipThreshold.notNil, {
 			o = o ++ " -s " ++ safetyClipThreshold;
 		});
+		if (Platform.hasBelaSupport, {
```

Comment:
`hasBelaSupport` is a primitive for Platform, like `hasQt`. Since remote servers can't be booted by a remote client, this part of the code is meant to be run by a client running on the Bela itself. That client has to be built with Bela support, so IMO it makes sense to check Platform and not ServerOptions or Server. But I'm open to alternatives :)